src/file.ts

```diff
@@ -57,7 +61,7 @@ export async function getFileData(filepath: string): Promise<FileInfo> {
 
     switch (fileInfo.fileType) {
         case 'png':
-            fileInfo.image = PNG.sync.read(fileInfo.buffer);
+            fileInfo.image = PNG.sync.read(fileInfo.buffer);          
```

Comment:
Can you delete the extra spaces at the end?

Reply:
fixed